### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by Nolski**
## Description
Updated our three READMEs to meet minimum standards for OpenMined projects

## How has this been tested?
- read it

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,59 @@
...
-[![npm downloads](https://img.shields.io/npm/dt/threepio.svg)](https://npm.im/threepio)
+![Tests](https://img.shields.io/github/workflow/status/OpenMined/threepio/Tests)
+![Coverage](https://img.shields.io/codecov/c/github/OpenMined/threepio)
+![License](https://img.shields.io/github/license/OpenMined/threepio)
+![OpenCollective](https://img.shields.io/opencollective/all/openmined)
 
-A multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js
+# Threepio
 
+Threepio is a multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js
```
Comment (cereallarceny) [inline README.md]: I love this description and the emoji list below. But tell me some **features** of your library, too (in emoji form). You can do it below this sentence and above the three main components section. Some ideas would be: bi-directional translation, argument and keyword argument reordering, multi-lingual, works between 3 frameworks, X% coverage for Y framework, support for fuzzy finding name matching (in progress), support for versioning (in progress), support for argument typecasting (in progress). Just some ideas!

File: README.md [hunk 2]
```diff
@@ -1,42 +1,59 @@
...
-```sh
-npm install --save threepio
+```bash
+pip install 3p0
 ```
 
+#### Usage
 
-## Usage
+Please check out our usage document for further guidelines.
```
Comment (cereallarceny) [inline README.md]: Link me bruh!

File: README.md [hunk 3]
```diff
@@ -1,42 +1,59 @@
...
 
-const threepio = new Threepio();
+```bash
+npm install @openmined/threepio
 ```
 
-## License
+#### Usage
+
+Please check out our usage document for further guidelines.
```
Comment (cereallarceny) [inline README.md]: Link me bruh!

File: pythreepio/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Threepio is a multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js 
+This section of the code-base is for the [Python Client](https://pypi.org/project/3p0/).
+
+
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install threepio.
+
+```bash
+pip install 3p0
```
Comment (cereallarceny) [inline pythreepio/README.md:17]: As much as I fucking love this package name, you will need to change it: https://github.com/OpenMined/.github/issues/11

File: README.md [hunk 5]
```diff
@@ -1,42 +1,59 @@
...
+Depending on your language, installation will differ.
 
-[npm][]:
+### Python
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install threepio.
 
-```sh
-npm install --save threepio
+```bash
+pip install 3p0
```
Comment (cereallarceny) [inline README.md:38]: See comment below about renaming this package name.

**Review by cereallarceny — CHANGES_REQUESTED** [review]
Just a few things!

File: pythreepio/README.md (hunk 4 above)
Comment (Nolski) [reply pythreepio/README.md:17]: Yup! I'll update the name in a PR down the line

**Review by cereallarceny — APPROVED** [review]
